### PR TITLE
trying to get tests to work again

### PR DESCRIPTION
**Pull request by AbdullahAlger**
For some reason, my work didn't get pushed into the project yesterday. Anyway, I had to make the test file again, which is ok. However, I am getting an error that I have been trying to fix for awhile and I do not see why there is an error. 

```
     1) User registers with valid details
     Failure/Error: expect(current_path).to eq "/"
       
       expected: "/"
            got: "/users"
       
       (compared using ==)
     # ./spec/features/users_register_spec.rb:18:in `block (2 levels) in <top (required)>'

```

I see that when I user registers, they are directed to the confirmation page within `/users/`, which seems to be the issue, I think. 

Any help would be appreciated.

### CODE REVIEW

File: spec/features/users_register_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    click_link "Register"
+    expect(current_path).to eq(new_user_registration_path)
+
+    fill_in "Name", with: "John Jacob"
+    fill_in "Email", with: "tester@example.tld"
+    fill_in "Password", with: "test-password"
+    fill_in "Password confirmation", with: "test-password"
+    click_button "Sign up"
+
+    expect(current_path).to eq "/"
```
Comment (eliotsykes) [inline spec/features/users_register_spec.rb]: Line 18 is where the stack trace reports the expectation failure.

When debugging these feature specs, its useful to be able to see the generated HTML of the current page.

On line 17, use one of these debugging techniques for seeing the current state of the page: https://github.com/jnicklas/capybara#debugging

Comment (AbdullahAlger) [inline spec/features/users_register_spec.rb]: Yep. I got 

```
file:///Users/Abdullah-Mac/code/ruby-projects/Blocipedia/tmp/capybara/capybara-201507171324524955565018.html
```

**Comment by AbdullahAlger** [comment]
Got the tests working now.

File: spec/features/users_register_spec.rb (hunk 1 above)
Comment (eliotsykes) [inline spec/features/users_register_spec.rb]: oh right I remember we hit that on the call!

**Pull request merged**